.github/CHANGELOG.md

```diff
@@ -465,6 +465,13 @@
 
 <h3>Improvements</h3>
 
+* The tape does not verify any more that all Observables have owners in the annotated queue.
+  [(#1505)](https://github.com/PennyLaneAI/pennylane/pull/1505)
+  This allows manipulation of Observables inside a tape context. An example is 
+  `expval(Tensor(qml.PauliX(0), qml.Identity(1)).prune())` which makes the expval an owner 
+  of the pruned tensor and its constituent observables, but leaves the original tensor in 
+  the queue without an owner.
```

Comment:
I'm not so familiar with the depths of PennyLane here; from trying out the examples, this leaves the tensors in the queue, but doesn't move them to `tape.operations`/`tape.measurements` or anything. Why do they need to be in the queue at all except when they are being used as part of the circuit?

Reply:
I am also just starting to understand queuing. Basically, from what I grasp, the queue is just memorising every operation that was recorded in the context. Technically after pruning some ops do not need to be there any more, and previously we have just manipulated and updated the queue - but apparently that was prone to error. So now we try not to touch the queue, but  record which operations are owners of others, and ultimately only the top-level owners get added to `tape.operations/tape.measurements`...